src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_MAX = 100;
+    private static final int RANDOM_MIN = 1;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random;
+
+    public Lottery() {
+        this.random = new Random();
+    }
```

Comment:
let's make your code consistent. Either initialize all instances in fields or within constructor

Reply:
done)